packages/databyss-ui/primitives/View/HoverView.js

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef } from 'react'
```

Comment:
Let's rename this `EditorTooltip` and put it in the `components/Editor` directory